support/src/main/java/se/inera/intyg/common/support/facade/util/ValueToolkit.java

```diff
@@ -74,6 +76,20 @@ public static String textValue(Map<String, CertificateDataElement> data, String
         return textDataValue.getText();
     }
 
+    public static String uncertainDateValue(Map<String, CertificateDataElement> data, String questionId, String valueId) {
+        final var dataValue = getValue(data, questionId);
+        if (!(dataValue instanceof CertificateDataUncertainDateValue)) {
```

Comment:
will this trigger if datevalue == null?

Reply:
Yes. If dateValue is null it won't be an instanceof CertificateDataValueDate and the whole statement will be true.